core/dbt/contracts/graph/semantic_manifest.py

```diff
@@ -59,8 +68,46 @@ def write_json_to_file(self, file_path: str):
         write_file(file_path, json)
 
     def _get_pydantic_semantic_manifest(self) -> PydanticSemanticManifest:
+        pydantic_time_spines: List[PydanticTimeSpine] = []
+        daily_time_spine: Optional[PydanticTimeSpine] = None
+        for node in self.manifest.nodes.values():
+            if not (isinstance(node, ModelNode) and node.time_spine):
+                continue
+            time_spine = node.time_spine
```

Comment:
Can `time_spine` be `None` at this point? If so would `standard_granularity_column` always be `None` as well and raise the `ParsingError`?

Reply:
The `continue` block above ensures that `time_spine` is not `None` at this point!